spec/services/copy_case/cloner_service_spec.rb

```diff
@@ -197,5 +240,15 @@
         expect(call).to be false
       end
     end
+
+    context "when cloning involved_children" do
```

Comment:
Should this have a suffix?
```suggestion
    context "when cloning involved_children raises an error" do
```